backend/packages/Upgrade/src/api/repositories/StateTimeLogsRepository.ts

```diff
@@ -0,0 +1,41 @@
+import { StateTimeLog } from '../models/StateTimeLogs';
+import { Repository, EntityRepository, EntityManager } from 'typeorm';
+import repositoryError from './utils/repositoryError';
+import { EXPERIMENT_STATE } from 'upgrade_types';
+import uuid from 'uuid';
+import { Experiment } from '../models/Experiment';
+
+@EntityRepository(StateTimeLog)
+export class StateTimeLogsRepository extends Repository<StateTimeLog> {
+  public async insertStateTimeLog(
```

Comment:
Instead of passing individual parameters create an entity of StateTimeLog and pass that.

Reply:
I have made an instance of the stateTimeLog class and initiated the values.